app/org/sagebionetworks/bridge/validators/SurveySaveValidator.java

```diff
@@ -224,18 +263,30 @@ private void validateOneRuleSet(Errors errors, SurveyRule rule, Set<String> alre
         if (rule.getAssignDataGroup() != null) {
             actionCount++;
         }
-        if (actionCount != 1) {
-            errors.rejectValue(fieldPath, "must have one and only one action: skipTo, endSurvey, or assignDataGroup");
+        if (rule.getDisplayIf() != null) {
+            actionCount++;
         }
-        if (rule.getAssignDataGroup() != null && !dataGroups.contains(rule.getAssignDataGroup())) {
-            errors.rejectValue(fieldPath, "has a data group '" + rule.getAssignDataGroup()
-                    + "' that is not a valid data group: " + BridgeUtils.COMMA_SPACE_JOINER.join(dataGroups));            
+        if (rule.getDisplayUnless() != null) {
+            actionCount++;
         }
```

Comment:
I've added one test for each action to verify the action makes a valid rule... I can alter the validator to return the count, then one test could verify that all actions are being counted.